src/module/implements/implementBenefits/lantern.js

```diff
@@ -93,73 +59,32 @@ class Lantern extends Implement {
         option: "lantern-implement-lit",
         toggleable: true,
       },
-      {
-        domain: "all",
-        key: "RollOption",
-        label: "Target Within Lantern Bright Light",
-        option: "target-in-lantern-bright-light",
-        toggleable: true,
-        predicate: ["lantern-implement-lit"],
-        hideIfDisabled: true,
-      },
       {
         key: "FlatModifier",
         selector: "perception",
-        label: "Lantern Bright Light Perception",
+        label: "In Lantern Light",
         type: "status",
         value: 1,
         predicate: [
           "lantern-implement-lit",
-          "target-in-lantern-bright-light",
+          "target:effect:in-lantern-light-enemy",
           { not: "check:type:initiative" },
         ],
-        slug: "lantern-per",
-        hideIfDisabled: true,
+        slug: "lantern-perception",
       },
       {
         key: "FlatModifier",
-        selector: "all",
-        label: "Lantern Bright Light Recall Knowledge",
-        type: "status",
+        selector: "skill-check",
+        label: "In Lantern Light",
         value: 1,
         predicate: [
           "lantern-implement-lit",
-          "target-in-lantern-bright-light",
+          "target:effect:in-lantern-light-enemy",
           "action:recall-knowledge",
-          { nor: ["adept:lantern"] },
+          { not: "check:statistic:perception" },
         ],
```

Comment:
These can be added into the ally aura if it's changed to `affects:{includesSelf:true}`, right? Is there a benefit to keeping the self effects separated? I'm afraid people will get confused when they see their allies getting the effect, but they don't.

Reply:
I believe it could be done, and I thought about doing that, but decided not to for a few reasons.

With EV target, IV, tome skill effect, etc. there are bunch of effects on the thaum.  Being in their own aura adds yet another.  But the implement already has some rules added, so putting them there has basically zero additional cost and gets rid of an effect on the thaum.

When the lantern is unlit, the bonuses won't exist at all (because no effect from the aura).  The IV effect will try to AdjustModifier them, but they won't exist.  With the bonuses on the implement, they still exist, but are off because the predicate is based on lantern lit.  So they can be adjusted.

Auras don't work in TotM play, without an active encounter and a gridded map.  But the bonuses still work if they are on the implement instead of an aura applied effect.

I did think about confusion, but there's two things that I think will make it ok.  One is the perception bonus still shows up in rolls, it'll just be disabled when it doesn't apply, like normal for the system.  So people will see the bonus there and if it's off, think about "is the lantern lit," or "did I target the creature?"

And the effect is named "Effect: In Lantern Light (Ally)"  So hopefully the "Ally" makes it clear it's for allies.

Reply:
Seems like valid enough reasons for me, glad we're at least on the same page with some of our concerns. I will make sure the wiki page specifies this.